aiohttp/web_urldispatcher.py

```diff
@@ -672,6 +672,54 @@ def __repr__(self):
             prefix=self._prefix, app=self._app)
 
 
+class SubDomainResource(AbstractResource):
+
+    def __init__(self, subdomain, app):
+        super().__init__()
+        self._subdomain = subdomain
```

Comment:
If we go for changing subdomain to domain, I guess it might be useful to use Yarl to parse the subdomain here so we get for free the port number and can actually dispatch on the actual host (domain + port number)

Reply:
Do you mean that we should patch yarl to parse the domain wiht subdomain?